e2e/e2e_test.go

```diff
@@ -976,13 +978,13 @@ func Test_Hedera_EVM_BurnERC721_Transfer(t *testing.T) {
 	expectedTxId := validateBurnERC721Event(burnTxReceipt, expectedRouterBurnERC721, t)
 
 	// 4. Validate that a scheduled NFT transaction was submitted
-	scheduledTxID, scheduleID := validateScheduledNftTransfer(setupEnv, expectedTxId, nftToken, serialNumber, t)
+	scheduledTxID, scheduleID := validateScheduledNftAllowanceApprove(t, setupEnv, expectedTxId, blockTimestamp.UnixNano())
 
 	// 5. Validate Event Transaction ID retrieved from Validator API
 	validateEventTransactionIDFromValidatorAPI(setupEnv, expectedTxId, scheduledTxID, t)
 
 	// 6. Validate that the NFT was sent to the receiver account
-	validateNftOwner(setupEnv, nftToken, serialNumber, setupEnv.Clients.Hedera.GetOperatorAccountID(), t)
+	validateNftSpender(t, setupEnv, nftToken, serialNumber, setupEnv.Clients.Hedera.GetOperatorAccountID())
 
```

Comment:
Last but not least, the user (e2e test) has to execute the crypto transfer transaction and take the nft to its account. The idea is e2e tests to be circular - previous test wraps the NFT, this one unlocks/unwraps it.